lib/livebook_web/live/session_live/secrets_live.ex

```diff
@@ -0,0 +1,70 @@
+defmodule LivebookWeb.SessionLive.SecretsLive do
+  use LivebookWeb, :live_view
```

Comment:
Is there a reason we need a new LV here?

Reply:
The form fields were resetting. Apparently on each memory update.

Reply:
I see, so we just need to make sure that we initialize the fields only on the first `update`. Something like this:

```elixir
@impl true
def mount(socket) do
  {:ok, assign(socket, initialized: false)}
end

@impl true
def update(assigns, socket) do
  socket = assign(socket, assigns)

  socket =
    if socket.assigns.initialized do
      socket
    else
      assign(socket, data: %{"label" => "", "value" => ""}, initialized: true)
    end

  {:ok, socket}
end
```

Reply:
Wait, we can just set `:data` on mount rather than `update`, no?

Reply:
Yup! Thanks ❤️